### DISCUSSION TITLE
Beets writing multiple values for TRACK and YEAR fields

### DISCUSSION THREAD

**Discussion by maniacalmaniac** [beetbox/beets#4995] [Q&A]
I'm just starting out using Beets, so this might be a "me" problem.  Whenever I import a new album into my beets library, a couple of strange things happen:

- TRACK metadata is duplicated.  There are 2 TRACK fields, each with the same information (1 and 1, 2 and 2, etc).  Is it possible to stop this?
- YEAR metadata has 2 values.  One is the year, the other is in year-month-day format (so basically a clone of ORIGINALDATE).  Is there any way to pick one or the other?

**Comment by sampsyo** [comment]
Hello! The current state of this problem is summarized here:
https://github.com/beetbox/beets/issues/350#issuecomment-228129982

To briefly summarize, MediaFile attempts to maximize compatibility by using multiple possible formatting of the same data. This is invisible to most tools, but some will report the duplicated information verbatim.
